docs/DemoInstructions.md

```diff
@@ -18,7 +18,7 @@ If you want, you can use your verified email credential to get an IIWBook creden
 
 ## Use the Verifiable Credential Identity Provider
 
-With the necessary verified credential(s) in hand, you are ready to access the protected resources&mdash;the mock BC Government service ("Access to Audio"). Go to the [service](https://csb-audio.pathfinder.gov.bc.ca/), and click on the "Verifiable Credential" authorization option. You will see a QR code that your mobile agent should be able to process. Assuming you have your verified email credential already, the mobile agent will help you present your credentials. If you only have the verified email credential, you will be prompted to manually enter your name, and on completion, you will have access to the service.
+With the necessary verified credential(s) in hand, you are ready to access the protected resources&mdash;the mock BC Government service ("Access to Audio"). Go to the [service](https://verified-email-authentication.pathfinder.gov.bc.ca), and click on the "Proceed" buttong to authenticate. You will see a QR code that your mobile agent should be able to process. Assuming you have your verified email credential already, the mobile agent will help you present your credentials and you will be granted access to the service.
```

Comment:
Typo in this "buttong" -- please fix.

Could we change path?  How about "mock-a2a.pathfinder.gov.bc.ca" or something like that.

Reply:
Typo fixed.

I can change the URL, however it isn't really a "mock A2A" anymore, it is just a portal where you can sign-in using the Verified Email credential - thatis why I went with the more generic `verified-email-authentication`.